contracts/test/PoolLiquidation.t.sol

```diff
@@ -23,28 +23,30 @@ contract PoolLiquidationTest is TestUtil {
         gov.setMaxSwapSlippage(2000);  // Set to 20% for the sake of the BPT shortfall test
     }
 
-    function test_triggerDefault_pool_delegate() public {
+    function test_triggerDefault_pool_delegate(uint256 totalMintedAmt) public {
```

Comment:
I feel like we can keep this as a unit test as well actually, feel pretty good about its assertion of minLoanEquity as is